templates/rhpam70-kieserver-externaldb.yaml

```diff
@@ -27,11 +27,11 @@ parameters:
 - displayName: Maven repository username
   description: Username to access the Maven repository, if required.
   name: MAVEN_REPO_USERNAME
-  required: false
+  required: true
```

Comment:
Why is username and password set as required here? There may be repositories which don't require authentication.